.tekton/kola-nfs-push.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     pipelinesascode.tekton.dev/cancel-in-progress: "false"
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
-      == "main"
+      == "main" && "./tests/containers/nfs/***".pathChanged()
```

Comment:
I think this is fine, though IIUC it suffers from a classic problem in the domain of change detection, which is that you do still want fresh content even if the definition files didn't change.

I guess it doesn't matter too much for e.g. these test containers, but it does more for cosa. See https://github.com/coreos/fedora-coreos-pipeline/blob/1a24c59c1e352572eff095cb8c8fc60fe8402afa/jobs/build-cosa.Jenkinsfile#L30.